data/ddl.sql

```diff
@@ -61,11 +61,15 @@ create table user_groups (
 create table rsvps (
     user_id uuid references  users(id),
     event_id uuid references events(id),
+    date timestamptz not null,
+    in_waitlist boolean not null,
     primary key (user_id, event_id)
 );
 
+alter table rsvps alter column in_waitlist set default FALSE;
+
 create table user_bans (
```

Comment:
Postgres enables inlining default values.  You can put this is inline in the create table statement: `in_waitlist boolean not null default false,`

Reply:
I'd also want to make sure w/ @QuincyLarson that wait listing is an MVP feature. I suspect that's its going to be easy to start bikeshedding the database before we actually have the API hooked up and using it (see various open PRs).

Reply:
done, thanks